registry/handlers/app.go

```diff
@@ -274,6 +276,28 @@ func (app *App) configureRedis(configuration *configuration.Configuration) {
 	}))
 }
 
+// configureMail prepares logging hook parameters.
+func (app *App) configureLogHook(configuration *configuration.Configuration) {
```

Comment:
This comment does not match the function name.  This should be caught by golint when you run 'make'